qiskit_experiments/calibration/experiments/rabi.py

```diff
@@ -0,0 +1,337 @@
+        )
+
+        if plot and plotting.HAS_MATPLOTLIB:
+            ax = plotting.plot_curve_fit(fit_fun, fit_result, ax=ax)
+            ax = plotting.plot_scatter(xdata, ydata, ax=ax)
+            self._format_plot(ax)
+            figures = [ax.get_figure()]
+        else:
+            figures = None
+
+        return best_fit, figures
+
+    @staticmethod
+    def _fit_quality(fit_freq: float, reduced_chisq: float, fit_freq_err: float):
+        """Method to check the quality of the fit.
+
+        A good fit has:
+            - Have a reduced chi-squared lower than three.
+            - More than a quarter of a full period.
+            - Less than 10 full periods.
+            - An error on the fit frequency lower than the fit frequency.
+        """
+
+        if (
+            reduced_chisq < 3
+            and np.pi / 2 < fit_freq < 10 * 2 * np.pi
+            and (fit_freq_err is None or (fit_freq_err < fit_freq))
+        ):
+            return "computer_good"
+
+        return "computer_bad"
+
+    @classmethod
+    def _format_plot(cls, ax):
+        """Format curve fit plot."""
+        ax.tick_params(labelsize=14)
+        ax.set_xlabel("Amplitude [arb. unit]", fontsize=16)
+        ax.set_ylabel("Signal [arb. unit.]", fontsize=16)
+        ax.grid(True)
+
+
+class Rabi(BaseExperiment):
+    """An experiment that scans the amplitude of a pulse to calibrate rotations between 0 and 1.
+
+    The circuits that are run have an RXGate with the pulse schedule attached to it through
+    the calibrations. The circuits are of the form:
+
+    .. parsed-literal::
+
+                   ┌─────────┐ ░ ┌─┐
+              q_0: ┤ RX(amp) ├─░─┤M├
+                   └─────────┘ ░ └╥┘
+        measure: 1/═══════════════╩═
+                                  0
+
+    """
+
+    __analysis_class__ = RabiAnalysis
+
+    @classmethod
+    def _default_run_options(cls) -> Options:
+        """Default options values for the experiment :meth:`run` method."""
+        return Options(
+            meas_level=MeasLevel.KERNELED,
+            meas_return="single",
+        )
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default values for the pulse if no schedule is given."""
+        return Options(
+            duration=160,
+            sigma=40,
+        )
+
+    def __init__(self, qubit: int, amplitudes: Optional[Union[List[float], np.array]] = None):
+        """Setup a Rabi experiment on the given qubit.
+
+        Args:
+            qubit: The qubit on which to run the Rabi experiment.
+            amplitudes: The amplitudes that will be scanned.
+        """
+        super().__init__([qubit])
+
+        if amplitudes is not None:
+            self._amplitudes = amplitudes
+        else:
+            self._amplitudes = np.linspace(-0.95, 0.95, 51)
+
+    def circuits(
+        self,
+        backend: Optional[Backend] = None,
+        schedule: Optional[ScheduleBlock] = None,
+        **circuit_options,
```

Comment:
This is based off of old API. See current version. It is no longer take **options here.
https://github.com/Qiskit/qiskit-experiments/blob/009af65fc2612f8a8f41be33e832ac3f6024fc6c/qiskit_experiments/base_experiment.py#L174

Reply:
Fixed here: 5d23300